Website/Providers/DataProviders/SqlDataProvider/09.04.00.SqlDataProvider

```diff
@@ -277,6 +277,30 @@ AS
 
 GO
 
+/* Issue #2762  - Make Default Module Action Menu Configurable */
+/***************************************************************/
+
+BEGIN
+
+	INSERT INTO {databaseOwner}[{objectQualifier}PortalSettings] (PortalID, SettingName, SettingValue, CreatedByUserID, CreatedOnDate, LastModifiedByUserID, LastModifiedOnDate)
```

Comment:
As @sleupold stated, I'd prefer we didn't have the SQL script and just allowed the default in code to be used in the majority of cases.

Reply:
Sure, I'll remove it and have some troubleshooting time on why it didn't fully work :)